docs/developers/releasing.rst

```diff
@@ -440,6 +441,16 @@ You can safely branch, if required, for a ``stable/`` branch before you tag.
     $ git tag -a 2.6.0 -m "Tag version 2.6.0"
     $ git push --tags
 
+If you branch you will want to update the :file:`README.rst` file so that it
+points correctly to branched versions of badges and documentation.  Review and
+test the actual links created, you don't need to commit everything.
+
+.. code-block:: console
+
+    $ ./setup.py update_readme -w
+    $ git diff README.rst
+    $ git commit README.rst
```

Comment:
Can we prioritise such comments based on... we'd like to get 2.8.0 out sometime this decade.  Seriously, these are releasing docs!